server/routes/api/auth.js

```diff
@@ -52,7 +54,9 @@ router.post('/forgot', async (req, res) => {
     from: process.env.PASSWORD_RESET_AUTH_EMAIL,
     to: user.email,
     subject: 'Forgot Password Request',
-    html: `<p>You are receiving this because you, or someone else, requested a password reset. Click <a href="http://localhost:${process.env.PORT}/reset/${token}">here</a> to finish resetting your password.</p>`,
+    html: `<p>You are receiving this because you, or someone else, requested a password reset. Click <a href="http://localhost:${
+      process.env.PORT
+    }/reset/${token}">here</a> to finish resetting your password.</p>`,
```

Comment:
`<a href="http://localhost:${process.env.PORT}/reset/${token}">` needs to be `<a href="http://localhost:${process.env.PORT}/api/auth/reset/${token}">`
That is my fault. apologies.

Reply:
Fixed